ui/media/js/main.js

```diff
@@ -954,6 +959,16 @@ function onTaskCompleted(task, reqBody, instance, outputContainer, stepUpdate) {
     if (isSoundEnabled()) {
         playSound()
     }
+    updateTitle()
+}
+
+function updateTitle() {
+    let img_remaining = [...document.querySelectorAll("div .imageTaskContainer").entries()].map(c => htmlTaskMap.get(c[1])).filter(task => task.isProcessing).map(task => task.numOutputsTotal - Math.max(0, (task.batchesDone - 1) * task.reqBody.num_outputs)).reduce((total, value) => total + value, 0);
```

Comment:
Can we break this into multiple lines please? So that it's clearer what each step does, for future developers to grasp the logic at a quick glance? :)

Reply:
I've changed it to multiple lines with names that should explain what each line is doing in [ec7553f](https://github.com/easydiffusion/easydiffusion/pull/1377/commits/ec7553fe526bb488485067226b4283499e6eef86)